adoc/retail_chap_formulas.adoc

```diff
@@ -0,0 +1,249 @@
+
+[NOTE]
+====
+There are many different configurations possible.
+This procedure outlines a simple, single-disk, three-partition configuration.
+Adjust it to suit your own environment.
+====
+
+
+.Procedure: Configuring the branch server with saltboot
+
+. In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab￼.
+. Check the [systemitem]``Bind`` formula, and click [btn]``Save``.
+. Navigate to the menu:Formulas[Bind] tab, and set these parameters for Zone1:
+* In the [guimenu]``Config`` section, select [systemitem]``Include Forwarders``.
+* In the [guimenu]``Name`` field, use the fully qualified domain name (FQDN) of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
+* In the [guimenu]``Type`` field, select [systemitem]``master``.
+. Click [btn]``Add item`` to save your changes.
+. Set these parameters for Zone2:
+* In the [guimenu]``Name`` field, use the reverse zone for the configured IP range (for example: [systemitem]``1.168.192.in-addr.arpa``).
+* In the [guimenu]``Type`` field, select [systemitem]``master``
+. In the [guimenu]``Available Zones`` section, use these parameters for Zone1:
+* In the [guimenu]``Name`` field, use the FQDN of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
+* In the [guimenu]``File`` field, type the name of your configuration file *TODO: Check this* (for example: [systemitem]``branch1.txt``).
+. In the [guimenu]``Start of Authority (SOA)`` section, use these parameters for Zone1:
+* In the [guimenu]``Nameserver (Ns)`` field, use the FQDN of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
+* In the [guimenu]``Contact`` field, use the email address for the domain administrator.
+* Leave all other fields as their default values.
+. In the [guimenu]``Records`` section, in subsection [guimenu]``A``, click on [btn]``Add Item`` and use these parameters to set up an A record for Zone1:
+* In the [guimenu]``Hostname`` field, use the hostname of the branch server (for example: [systemitem]``branchserver``).
+* In the [guimenu]``IP`` field, use the IP address of the branch server (for example, [systemitem]``192.168.1.1``).
+. In the [guimenu]``Records`` section, subsection [guimenu]``NS``, click on [btn]``Add Item`` and use these parameters to set up an NS record for Zone1:
+* In the input box, use the hostname of the branch server (for example: [systemitem]``branchserver``).
+. In the [guimenu]``Records`` section, subsection [guimenu]``CNAME``, click on [btn]``Add Item`` and add the hostname of the branch server in each of these fields:
+* [systemitem]``tftp``
+* [systemitem]``ftp``
+* [systemitem]``dns``
+* [systemitem]``dhcp``
+* [systemitem]``salt``
+. Set up Zone2 using the same parameters as for Zone1, but ensure you use the reverse  details.
+. Click [btn]``Save Formula`` to save your configuration
+. Apply the highstate.
+
+
```

Comment:
Here should be info from https://github.com/SUSE/spacewalk/issues/6087:

> [NOTE]
> ====
> If you use networks inside one of the IPv4 private address ranges:
> * 10.0.0.0/8
> * 172.16.0.0/12
> * 192.168.0.0/16
> then reverse name resolution on terminals might not work. If that is the case, the solution is to press button `Add item` inside `Options` section of the `bind` formula, and the to enter `empty-zones-enable` in the `Options:` field and `no` in the `Value:` field.
> ====

Reply:
This procedure has been removed as per your earlier comment.